File: src/main/java/com/github/zxh/classpy/classfile/datatype/Bytes.java
package com.github.zxh.classpy.classfile.datatype;

import com.github.zxh.classpy.classfile.ClassFileComponent;
import com.github.zxh.classpy.classfile.ClassFileReader;

/**
 * Unparsed bytes.
 */
public class Bytes extends ClassFileComponent {

    private UInt count;

    public Bytes(UInt count) {
        this.count = count;
    }

    @Override
    protected void readContent(ClassFileReader reader) {
        reader.skipBytes(count.getValue());
    }

}

File: src/main/java/com/github/zxh/classpy/classfile/datatype/Bytes.kt
package com.github.zxh.classpy.classfile.datatype

import com.github.zxh.classpy.classfile.ClassComponent
import com.github.zxh.classpy.classfile.reader.ClassReader

/**
 * Unparsed bytes.
 */
class Bytes(val count: IntType) : ClassComponent() {

    override fun readContent(reader: ClassReader) {
        reader.skipBytes(count.value);
    }

}
